assignments/objects.js

```diff
@@ -11,45 +11,90 @@
 
 // Example format of an intern object: 1,examples@you.edu,Example,F
 let example = {
-  "id": 0,
-  "name": "Example",
-  "email": "examples@you.edu",
-  "gender": "F"
-}
+  id: 0,
+  name: "Example",
+  email: "examples@you.edu",
+  gender: "F"
+};
 
 // Write your intern objects here:
 
-
-// ==== Challenge 2: Reading Object Data ==== 
+let interns = [
+  {
+    id: 0,
+    name: "Mitzi",
+    email: "mmelloy0@psu.edu",
+    gender: "F"
+  },
+  {
+    id: 1,
+    name: "Kennan",
+    email: "kdiben1@tinypic.com",
+    gender: "M",
+    speak: function() {
+      return "Hello, my name is Kennan!";
+    }
+  },
+  {
+    id: 2,
+    name: "Kevin",
+    email: "kmummery2@wikimedia.org",
+    gender: "M"
+  },
+  {
+    id: 3,
+    name: "Gaini",
+    email: "gmartinson3@illinois.edu",
+    gender: "F"
+  },
+  {
+    id: 4,
+    name: "Antoinetta",
+    email: "adaine5@samsung.com",
+    gender: "F",
+    multiplyNums: function(a, b) {
+      return a * b;
+    }
+  }
+];
+
```

Comment:
Check the solution code for an alternate solution. He creates individual objects stored in seperate variables.
```
let kevin = {};
``` 
for example.